central_v2/src/lib/CentralModels.tsx

```diff
@@ -1,5 +1,11 @@
 import { CentralQuestionTemplateInput, IncorrectCard } from "@righton/networking";
 
+export enum UserStatusType {
```

Comment:
Defines all possible states user can be in over course of app (so we can handle them consistently)